roles/redhat_subscription/tasks/main.yml

```diff
@@ -21,14 +21,22 @@
 # Below we use string formatting to use a variable as the name of the file in
 # the lookup()
 # via http://stackoverflow.com/a/34239020
-#
-  - name: Get origin file
-    command: ostree admin --print-current-dir
-    register: origin_file
 
-  - name: Get refspec from origin file
-    command: grep refspec {{ origin_file.stdout }}.origin
-    register: refspec
+  - name: status of /usr/bin/ostree is known
+    shell: type -Pf /usr/bin/ostree | /bin/true
```

Comment:
Well, trying that particular command on a F25 Cloud system blows up and returns `1`, so that would cause the role to fail and kill the whole playbook.

The role is definitely coupled to Atomic Host, because we didn't consider running these tests on anything but an Atomic Host.  😎 

I believe the reason we are manipulating the `.origin` file here is actually a side effect of how we provision systems in our Jenkins jobs.  Without getting into those details, when a system gets registered via sub-man, that refspec can get clobbered.  @mike-nguyen can correct me here, if I'm wrong.

In this case, I'm not in favor of splitting out the Atomic-specific stuff because it would amount to only a task or two per role.